src/cta/index.js

```diff
@@ -41,8 +41,7 @@ if (typeof discovery_response !== 'function') {
 
 document.getElementById('main').style.background = window.xprops.backgroundColor;
 document.getElementById('idpbutton').style.background = window.xprops.color;
-document.getElementById('idpbutton').style.color = window.xprops.buttonTextColor;
-document.getElementById('dsbutton').style.color = window.xprops.linkTextColor;
+document.getElementById('dsbutton').style.color = window.xprops.color;
```

Comment:
The same color is applied to the background and the link below is one parameter "color", which already existed in the past.